src/derive/curve.rs

```diff
@@ -279,12 +275,12 @@ macro_rules! new_curve_impl {
 
                         fn from_uncompressed_unchecked(bytes: &Self::Uncompressed) -> CtOption<Self> {
                             let bytes = &bytes.0;
-                            let infinity_flag_set = if $spare_bits == 1 {
```

Comment:
Hmm, how about `identity_flat_set`, instead of `infinity_flag_set`? @davidnevadoc 
I think the word `identity` is used everywhere in proposal and code.